code/game/objects/structures/flora.dm

```diff
@@ -16,7 +16,7 @@
 	icon_state = "pine_1"
 
 /obj/structure/flora/tree/pine/Initialize(mapload)
-	. = ..()
```

Comment:
This actually breaks things. Undo this change.

Reply:
Done, probably should put more thinking into those changes.